test/test.spec.ts

```diff
@@ -94,6 +95,20 @@ describe('service', function () {
       // Assert
       expect(componentInstance).toEqual(componentInstance2);
     })
+
```

Comment:
This looks exactly like the report embed test.  It seems like this was added to test dashbaord embedding but the test is actually only intended to test the specific `get` method on the service and repeating it shouldn't be needed.  Needing to a test for each type of embed is probably an indicator that these tests aren't written correctly.

I think this tests are in weird spot where they are attempting to be unit tests on the service, but end up behaving like integration tests.  I would say since changing the test design is probably a larger problem we can leave this one as is even though it appears redundant and try to refactor later.